### DISCUSSION TITLE
Angular latest version

### DISCUSSION THREAD

**Discussion by sfjadi2010** [jasontaylordev/CleanArchitecture#1068] [General]
How to upgrade to angular latest version?

**Comment by Tony20221** [comment]
npm uninstall -g @angular/cli
npm install -g @angular/cli@latest

**Reply by jaresus** [reply]
I did so (update), I changed the components to standalone, but I kept having problems redirecting to the login page when I was not authenticated. 
I couldn't fix it, and I really want them to be standalone.
:(

**Reply by deepak-anmol** [reply]
I am able to convert successfully to standalone components as well as to latest angular version 20.
To do this, there is long process of migration and conversion process to standalone components
